packages/client/src/components/Actions/components/TokenView.js

```diff
@@ -0,0 +1,66 @@
+import { useClipboard, useContacts } from "hooks";
+import Svg from "library/Svg";
+import { getSafeContactName, getTokenMeta } from "utils";
+import ProposedDateView from "./ProposedDateView";
+
+const TokenView = ({ actionView, safeData }) => {
+  const { recipient, vaultId, tokenAmount, timestamp } = actionView;
+  const { name: safeName, address: safeAddress } = safeData;
+
+  const { displayName, tokenType } = getTokenMeta(vaultId) || {};
+
+  const clipboard = useClipboard();
+  const { contacts } = useContacts(safeAddress);
+
+  return (
+    <>
+      <div className="pl-4 mb-5">
+        <span className="columns">Amount</span>
+        <span className="columns is-size-2 is-family-monospace has-text-black">
+          {Number(tokenAmount)}
+        </span>
+        <span className="columns is-size-6 has-text-weight-bold has-text-black">
+          <Svg name={tokenType} /> &nbsp; {displayName}
+        </span>
+      </div>
+      <div className="mt-4">
+        <ProposedDateView timestamp={timestamp} />
+        <div className="column is-vcentered is-multiline is-mobile border-light-bottom is-flex is-full">
```

Comment:
Do we intend to have this div as a container or a child element or both?

Reply:
Can you please elaborate?

Reply:
`is-vcentered`, `is-multiline` and `is-mobile` are modifiers for the columns class, they need to be used in conjunction to the columns class(container class) but I'm seeing `column`, which is children element class, that's why I'm confused